streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -171,6 +172,9 @@ public void makeReady(final Map<String, InternalTopicConfig> topics) {
      */
     // visible for testing
     protected Map<String, Integer> getNumPartitions(final Set<String> topics) {
+        log.debug("Trying to check if topics {} have been created with expected number of partitions.", topics);
+
+        // TODO: KAFKA-6928. should not need retries in the outer caller as it will be retried internally in admin client
```

Comment:
Let's avoid TODOs in the code. We can point to the relevant part of the code in the JIRA.